cpp/tests/sampling/sg_uniform_neighbor_sampling.cu

```diff
@@ -128,25 +128,29 @@ class Tests_Uniform_Neighbor_Sampling
                random_sources.size(),
                handle.get_stream());
 
-#ifdef NO_CUGRAPH_OPS
-    EXPECT_THROW(cugraph::uniform_neighbor_sample(
-                   handle,
-                   graph_view,
-                   edge_weight_view,
-                   std::nullopt,
-                   std::nullopt,
-                   std::move(random_sources_copy),
-                   std::move(batch_number),
-                   raft::host_span<int32_t const>(uniform_neighbor_sampling_usecase.fanout.data(),
-                                                  uniform_neighbor_sampling_usecase.fanout.size()),
-                   rng_state,
-                   true,
-                   uniform_neighbor_sampling_usecase.flag_replacement),
-                 std::exception);
-#else
     std::optional<std::tuple<raft::device_span<int32_t const>, raft::device_span<int32_t const>>>
       label_to_output_comm_rank_mapping{std::nullopt};
 
+#ifdef NO_CUGRAPH_OPS
+    EXPECT_THROW(
+      cugraph::uniform_neighbor_sample(
```

Comment:
Just FYI, this is one way to fix this, another is to call this function with explicit template types.

i.e. `cugraph::uniform_neighbor_sample<explicit template types...>(...);`

This might be less verbose in some cases.